app/server/lib/MinIOExternalStorage.ts

```diff
@@ -111,7 +132,7 @@ export class MinIOExternalStorage implements ExternalStorage {
 
   public async hasVersioning(): Promise<Boolean> {
     const versioning = await this._s3.getBucketVersioning(this.bucket);
-    return versioning && versioning.Status === 'Enabled';
+    return versioning !== "" && versioning && versioning.Status === 'Enabled';
```

Comment:
I read [this thread](https://github.com/gristlabs/grist-core/pull/991/files#r1611941032), it took me a little while to understand the reason of this code.

Could you add some comment to add more context? :pray:

Reply:
Comment added.